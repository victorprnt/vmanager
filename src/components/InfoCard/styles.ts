import styled from 'styled-components'

export const Wrapper = styled.div`
  display: flex;
  justify-content: space-between;
  align-items: center;

  width: 100%;
  padding: 1rem 2rem;

  background-color: #fff;
  border-radius: 6px;

  img {
    color: #000;
  }

  .infocard-text {
    color: #000;
    text-align: right;

    .infocard-value {
      font-size: 3rem;
    }
    .infocard-label {
      font-size: 1rem;
    }
  }
`
