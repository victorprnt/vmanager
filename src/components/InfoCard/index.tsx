import * as S from './styles'

const InfoCard = () => (
  <S.Wrapper>
    <img src="" alt="Infocard Image" className="infocard" />
    <div className="infocard-text">
      <p className="infocard-value">Value</p>
      <p className="infocard-label">Label</p>
    </div>
  </S.Wrapper>
)

export default InfoCard
