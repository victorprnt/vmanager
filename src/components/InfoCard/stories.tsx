// import { Story, Meta } from '@storybook/react'
import InfoCard from '.'

// export default {
//   title: 'InfoCard',
//   component: InfoCard
// } as Meta

// export const Default: Story = () => <InfoCard />
