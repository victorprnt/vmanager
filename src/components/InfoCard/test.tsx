import { render, screen } from '@testing-library/react'

import InfoCard from '.'

const props = {
  label: 'Assets',
  value: 33,
  total: 33
}

// describe('<InfoCard />', () => {
//   it('should render the heading', () => {
//     const { container } = render(<InfoCard {...props} />)

//     expect(
//       screen.getByRole('img', { name: /InfoCard Image/i })
//     ).toBeInTheDocument()
//     expect(screen.getByText(/value/i)).toBeInTheDocument()
//     expect(screen.getByText(/label/i)).toBeInTheDocument()

//     expect(container.firstChild).toMatchSnapshot()
//   })
// })
