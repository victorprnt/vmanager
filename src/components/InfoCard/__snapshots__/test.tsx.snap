// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<InfoCard /> should render the heading 1`] = `
<div
  class="sc-bczRLJ euDIkm infocard"
>
  <img
    alt="Infocard Image"
    class="infocard"
    src=""
  />
  <div
    class="infocard-text"
  >
    <p
      class="infocard-value"
    >
      Value
    </p>
    <p
      class="infocard-label"
    >
      Label
    </p>
  </div>
</div>
`;
